### PR TITLE
[Bluetooth][Non-ACR] Add public constructor for BluetoothDevice

### PR DESCRIPTION
**Pull request by Wootak**

### CODE REVIEW

**Review by dh79pyun — APPROVED** [review]
Looks good

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs [hunk 1]
```diff
@@ -62,6 +63,16 @@ internal BluetoothDevice()
         {
         }
 
+        /// <summary>
+        /// The constructor
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public BluetoothDevice(string address)
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs:71]: What is your intent? as I know, `BluetoothDevice` instance could not create by user, It only can get from `BluetoothAdaptor`

Comment (myroot) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs:71]: https://github.com/Samsung/TizenFX/blob/91e9650877281eb3d2e31de7ea12a1e7afdfbe9b/src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapter.cs#L519

Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs:71]: In now, BluetoothDevice instance created from StartDiscovery and DiscoveryStateChanged internally as you said.
But, there is one requirement to create bond by using StartLeScan. and, this scenarios is reasonable and possible in native.

Comment (myroot) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs:71]: Could you give more detail scenario? 
When `BluetoothAdapter`.`StartLeScan()` was used, user can get `BluetoothLeDevice` from `ScanResultChanged` event
Why need to create `BluetoothDevice`?

Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs:71]: user need to call CreateBond() method. then, user need to create BluetoothDevice instance.

Comment (myroot) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs]: I recommend to add `CreateBond` method into `BluetoothLeDevice`

Comment (myroot) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs]:
``` c#
        TaskCompletionSource<BluetoothDevice> _createTask;
        BluetoothDevice _bluetoothDevice;

        public Task<BluetoothDevice> CreateBound()
        {
            if (_createTask != null)
            {
                return _createTask.Task;
            }

            _createTask = new TaskCompletionSource<BluetoothDevice>();
            _bluetoothDevice = new BluetoothDevice();
            _bluetoothDevice.RemoteDeviceAddress = this.RemoteAddress;

            _bluetoothDevice.BondCreated += (s, e) =>
            {
                if (e.Result == BluetoothError.None)
                {
                    _createTask.SetResult(e.Device);
                }
                else
                {
                    _createTask.SetException(BluetoothErrorFactory.CreateBluetoothException((int)e.Result));
                }
                _createTask = null;
            };
            try
            {
                _bluetoothDevice.CreateBond();
            }
            catch
            {
                _createTask = null;
                throw;
            }
            return _createTask.Task;
        }

```

**Review by myroot — APPROVED** [review]

**Review by dh79pyun — APPROVED** [review]
Looks good

**Pull request merged**